src/c++/perf_analyzer/infer_context.cc

```diff
@@ -139,6 +146,11 @@ InferContext::SendRequest(const uint64_t request_id, const bool delayed)
     thread_stat_->idle_timer.Start();
     start_time_sync = std::chrono::system_clock::now();
     cb::InferResult* results = nullptr;
+    {
+      std::lock_guard<std::mutex> lock(thread_stat_->mu_);
+      thread_stat_->request_send_times_.push_back(
```

Comment:
Does the client stat and/or the triton stats have a request count? Could we use that instead?
Also, does batch size need to be factored in anywhere? I think not with the method you are using, but it may be needed if we use triton stats

Reply:
Similar to [this response](https://github.com/triton-inference-server/client/pull/255#issuecomment-1447749252), the client stats and triton stats both are incremented when a request completes (the response is received/sent). I don't want the send request rate metric to be dependent on requests returning within the window.